frontend/src/pages/team/Devices/dialogs/DeviceCredentialsDialog.vue

```diff
@@ -14,13 +14,53 @@
-                        <div class="flex flex-row justify-end space-x-2 -mt-1">
-                            <ff-button kind="tertiary" size="small" @click="copy('npm install -g @flowfuse/device-agent')">
-                                <template #icon-right><ClipboardCopyIcon /></template>
-                                <span class="">Copy</span>
-                            </ff-button>
+                        <div class="mb-4">
+                            <ul class="grid grid-cols-3 space-x-2">
+                                <li
+                                    v-for="os in ['Windows', 'MacOS', 'Linux']"
+                                    :key="os"
+                                    class="p-4 mb-2 border rounded bg-white border-gray-200 cursor-pointer flex items-center justify-center space-x-2"
+                                    :class="{ 'border-blue-600': selectedOS === os }"
+                                    @click.stop="selectedOS = os"
+                                >
+                                    <img :src="osIcons[os]" class="w-5 h-5" :alt="os + ' icon'">
+                                    <span>{{ os }}</span>
+                                </li>
+                            </ul>
+                            <div class="grid grid-cols-1 gap-4">
+                                <div class="p-2 border rounded bg-gray-50">
+                                    <template v-if="selectedOS === 'Windows'">
+                                        <p>Open Command Prompt or PowerShell as administrator and run:</p>
+                                        <pre class="overflow-auto text-xs font-light p-4 my-2 border rounded bg-gray-800 text-gray-200">npm install -g @flowfuse/device-agent</pre>
+                                        <div class="flex flex-row justify-end space-x-2 -mt-1">
+                                            <ff-button kind="tertiary" size="small" @click="copy('npm install -g @flowfuse/device-agent')">
+                                                <template #icon-right><ClipboardCopyIcon /></template>
+                                                <span class="">Copy</span>
+                                            </ff-button>
+                                        </div>
+                                    </template>
+                                    <template v-if="selectedOS === 'MacOS'">
+                                        <p>Open Terminal and run:</p>
+                                        <pre class="overflow-auto text-xs font-light p-4 my-2 border rounded bg-gray-800 text-gray-200">npm install -g @flowfuse/device-agent</pre>
+                                        <div class="flex flex-row justify-end space-x-2 -mt-1">
+                                            <ff-button kind="tertiary" size="small" @click="copy('npm install -g @flowfuse/device-agent')">
```

Comment:
```suggestion
                                        <pre class="overflow-auto text-xs font-light p-4 my-2 border rounded bg-gray-800 text-gray-200">sudo npm install -g @flowfuse/device-agent</pre>
                                        <div class="flex flex-row justify-end space-x-2 -mt-1">
                                            <ff-button kind="tertiary" size="small" @click="copy('sudo npm install -g @flowfuse/device-agent')">
```
Joe, from my reading (and [our docs](https://flowfuse.com/docs/device-agent/install/)) it seems MacOS also needs `sudo`

Please check and apply if true.